tests/format-bar.spec.ts

```diff
@@ -1650,3 +1631,65 @@ test('create linked doc from block selection with format bar', async ({
 </affine:page>`
   );
 });
+
+test.describe('more menu button', () => {
+  test('should be able to perform the copy action', async ({ page }) => {
+    await enterPlaygroundRoom(page);
+    await initEmptyParagraphState(page);
+    await initThreeParagraphs(page);
+    // drag only the `456` paragraph
+    await dragBetweenIndices(page, [1, 0], [1, 3]);
+
+    const { openMoreMenu, copyBtn } = getFormatBar(page);
+    await openMoreMenu();
+    await expect(copyBtn).toBeVisible();
+    await assertRichTextInlineRange(page, 1, 0, 3);
+    await copyBtn.click();
+    await assertRichTextInlineRange(page, 1, 0, 3);
+
+    await focusRichText(page, 1);
+    await pasteByKeyboard(page);
+    await waitNextFrame(page);
+
+    await assertRichTexts(page, ['123', '456456', '789']);
+  });
+
+  // fixme: Copy and paste is broken(ctrl-c/ctrl-v).
+  test.skip('should be able to perform the duplicate action', async ({
```

Comment:
After https://github.com/toeverything/blocksuite/pull/7386, Copy and paste is broken. /cc @fourdim

Reply:
### Before

https://github.com/toeverything/blocksuite/assets/27926/f48e9ae6-c414-4743-bd67-88a01d76d6c3


### After

https://github.com/toeverything/blocksuite/assets/27926/e00c8081-0094-4159-830b-a80290d39881

Reply:
@fourdim PTAL